readthedocs/projects/static-src/projects/js/import.js

```diff
@@ -282,6 +282,9 @@ function ProjectImportView(instance, config) {
             .then(function (data) {
                 self.get_organizations();
                 self.get_accounts();
+                // This will trigger a new
+                // request to re-sync the repos
+                self.filter_by.valueHasMutated();
```

Comment:
I took this from https://stackoverflow.com/questions/16261689/force-knockout-to-mark-an-observable-as-changed-even-if-focus-is-still-in-the-f#16261761, still not sure if this is the correct way, but I tested and works.

Reply:
This change looks better.